_api-reference/cluster-health.md

```diff
@@ -105,4 +105,112 @@ The following table lists all response fields.
+  "number_of_data_nodes": 3,
+  "discovered_master": true,
+  "discovered_cluster_manager": true,
+  "active_primary_shards": 0,
+  "active_shards": 0,
+  "relocating_shards": 0,
+  "initializing_shards": 0,
+  "unassigned_shards": 0,
+  "delayed_unassigned_shards": 0,
+  "number_of_pending_tasks": 0,
+  "number_of_in_flight_fetch": 0,
+  "task_max_waiting_in_queue_millis": 0,
+  "active_shards_percent_as_number": 100,
+  "awareness_attributes": {
+    "zone": {
+      "zone-3": {
+        "active_shards": 0,
+        "initializing_shards": 0,
+        "relocating_shards": 0,
+        "unassigned_shards": 0,
+        "data_nodes": 1,
+        "weight": 1
+      },
+      "zone-1": {
+        "active_shards": 0,
+        "initializing_shards": 0,
+        "relocating_shards": 0,
+        "unassigned_shards": 0,
+        "data_nodes": 1,
+        "weight": 1
+      },
+      "zone-2": {
+        "active_shards": 0,
+        "initializing_shards": 0,
+        "relocating_shards": 0,
+        "unassigned_shards": 0,
+        "data_nodes": 1,
+        "weight": 1
+      }
+    },
+    "rack": {
+      "rack-3": {
+        "active_shards": 0,
+        "initializing_shards": 0,
+        "relocating_shards": 0,
+        "unassigned_shards": 0,
+        "data_nodes": 1,
+        "weight": 1
+      },
+      "rack-1": {
+        "active_shards": 0,
+        "initializing_shards": 0,
+        "relocating_shards": 0,
+        "unassigned_shards": 0,
+        "data_nodes": 1,
+        "weight": 1
+      },
+      "rack-2": {
+        "active_shards": 0,
+        "initializing_shards": 0,
+        "relocating_shards": 0,
+        "unassigned_shards": 0,
+        "data_nodes": 1,
+        "weight": 1
+      }
+    }
+  }
+}
+```
+
+The `awareness_attributes` object contains cluster health information for each awareness attribute.
+
+Refer to the [Response fields section](#response-fields) for a description of the response fields. The `weight` field refers to the awareness attribute weight. 
+
+If you're interested in a particular awareness attribute, you can include the name of the awareness attribute as a query parameter:
+
+```json
+GET _cluster/health?level=awareness_attributes&awareness_attribute=zone
+```
+
+In response to the preceding request, OpenSearch returns cluster health information only for the `zone` awareness attribute.
+
+Cluster health by awareness attribute returns accurate information about unassigned shards only if you [enable replica count enforcement]({{site.url}}{{site.baseurl}}/opensearch/cluster#forced-replica-count-enforcement) and [configure forced awareness]({{site.url}}{{site.baseurl}}/opensearch/cluster#forced-awareness) for the awareness attribute either before cluster start or after cluster start but before any indexing requests. If you enable replica enforcement after the cluster receives indexing requests, unassigned shard information may be inaccurate. In this case, the `unassigned_shards` field will contain -1.
```

Comment:
I know we try to get away from using the future "will". But in this sentence it helps with clarity (will + if form a conditional phrase).

"Cluster health by awareness attribute will return accurate information about unassigned shards only if you [enable replica count enforcement] and [configure forced awareness] ..."

and then ...
 
"... for the awareness attribute before any indexing requests. This is effective both before or after you start the cluster."

Does this preserve the meaning?

Reply:
I think "will" does not add meaning here.  And this sentence does not preserve the meaning, because there are 2 cases where chbaa returns correct info about unassigned shards:
1. if you set up erce and configure fe before the cluster starts
2. if you set up erce and configure fe after the cluster starts but before it gets any indexing requests
If you do it after indexing requests, then chbaa may return incorrect results.

Reply:
RE: "will." It's a grammatical convention that signals a conditional situation: "I will go to the party if Peter stays home". "If it rains any more today, the river will definitely flood." But this is technical writing. We do tend to flatten things as much as possible. Agreed, it's fine as is.

How about, "Cluster health by awareness attribute returns accurate information about unassigned shards under either of two conditions: 1) if you set up erce and configure fe after the cluster starts but before it gets an indexing request, and 2) if you set up erce and configure fe after the cluster starts but before it gets an indexing request—If you do it after indexing requests, then chbaa may return incorrect results.

Reply:
This works but becomes a lot of text when you expand all the acronyms. Let's see what @natebower says.